lib/iris/aux_factory.py

```diff
@@ -307,31 +308,6 @@ def _remap_with_bounds(self, dependency_dims, derived_dims):
             nd_values_by_key[key] = nd_values
         return nd_values_by_key
 
-    def _shape(self, nd_values_by_key):
```

Comment:
@pp-mo This was only used in `OceanSigmaZFactory.make_coord()`, so you must have refactored that away ...